packages/manager/apps/pci-databases-analytics/src/lib/durationHelper.ts

```diff
@@ -63,3 +63,33 @@ export function durationToISODurationString(durationTime: Duration) {
 export function convertDurationStringToISODuration(durationTime: string) {
   return durationToISODurationString(durationStringToDuration(durationTime));
 }
+
+export function durationStringToSeconds(durationString: string) {
```

Comment:
As it is a function that computes a lot of elements, I'll suggest to add a test suite to ensure everything works as expected

Reply:
can you improve that with a loop ?

Reply:
Yep, both are fixed :) Thanks